### PR TITLE
Identity versions Update June 2021

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 1]
```diff
@@ -512,6 +512,16 @@
         "old": "field com.azure.core.amqp.exception.AmqpErrorContext.serialVersionUID",
         "new": "field com.azure.core.amqp.exception.AmqpErrorContext.serialVersionUID",
         "justification": "The field ErrorInfo was added to AmqpErrorContext, but no existing fields were removed or changed."
+      },
+      {
+          "code": "java.method.added",
+          "new": "method com.azure.core.util.HttpClientOptions com.azure.core.util.HttpClientOptions::setReadTimeout(java.time.Duration)",
+          "justification": "New Setter method."
```
Comment (srnagar) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:519]: Does azure-identity expose `ClientOptions` in any of its builders?

Comment (g2vinay) [reply eng/code-quality-reports/src/main/resources/revapi/revapi.json:519]: nope.

**Review by srnagar — APPROVED** [review]

**Pull request merged**